DEVELOPER_GUIDE.md

```diff
@@ -556,6 +556,39 @@ Then, you need to apply patterns for git-secrets, you can install the AWS standa
 git secrets --register-aws
 ```
 
+#### Run remote backed storage from dev environment
+
+To run the remote backed storage from the dev/local environment - Please do add the below to run.gradle file.
+
+```
+testClusters {
+   runTask {
+    // Add following lines to enable remote store
+    setting 'node.attr.remote_store.segment.repository', '<repository_name>'
+    setting 'node.attr.remote_store.translog.repository', '<repository_name>'
+    setting 'node.attr.remote_store.state.repository', '<repository_name>'
+    setting 'node.attr.remote_store.repository.my-repository.type', '<type>'
+    setting 'node.attr.remote_store.repository.my-repository.settings.bucket', '<bucket_name>'
+    setting 'node.attr.remote_store.repository.my-repository.settings.base_path', '<base_path>'
+    setting 'node.attr.remote_store.repository.my-repository.settings.region', '<region>'
+
+
+    // Add AWS credentials to the keystore
+    keystore 's3.client.default.access_key', '<access_key>'
```

Comment:
I don't think storing credentials in this file should be a recommended practice. 

How about reading from env variables instead as an alternative unless there are better ways:

```
keystore 's3.client.default.access_key', System.getenv('AWS_ACCESS_KEY_ID')
keystore 's3.client.default.secret_key', System.getenv('AWS_SECRET_ACCESS_KEY')
keystore 's3.client.default.session_token', System.getenv('AWS_SESSION_TOKEN')
```

```
export AWS_ACCESS_KEY_ID=<access_key>
export AWS_SECRET_ACCESS_KEY=<secret_key>
export AWS_SESSION_TOKEN=<session_token>

```

In that way onlly authorized users are able to run this script.

Reply:
I understand that storing credentials directly isn't the recommended approach, and I agree. You can definitely export these values and access them through system variables. My goal here is to demonstrate how to run OpenSearch locally while connecting to remote storage. 

The reason I’m using them directly in the file is that my tokens [access_key, secret, token] are temporarily generated and have a validity of only 45 minutes. 

Made the recommended changes.